src/material-experimental/mdc-autocomplete/autocomplete.html

```diff
@@ -6,6 +6,7 @@
     [ngClass]="_classList"
     [attr.aria-label]="ariaLabel || null"
     [attr.aria-labelledby]="_getPanelAriaLabelledby(formFieldId)"
+    [@panelAnimation]="isOpen ? 'visible' : 'hidden'"
```

Comment:
nit: Calling it "*Animation" is redundant since the `@` means that it's an animation. We could simplify it to just "panel"?